mobile/src/pods/CreatePod.tsx

```diff
@@ -24,13 +24,21 @@ interface Props {
 
 const CreatePod: React.FC<Props> = ({ navigation, route }) => {
   const [pod, setPod] = useState<Pod>();
+  const [invitees, setInvitees] = useState([]);
+
+  React.useEffect(() => {
+    if (route.params?.invitees) {
```

Comment:
What does the question mark operator do here?

Reply:
it checks that params are defined

Reply:
kinda like 
if (route.params) {
   if (route.params.invitees) {
     // code
   }
}